src/types/volunteer.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from 'zod';
+// import zEvents from './events'; <- import if made
+// import zHours from './hours'; <- import if made
+// import zDemographic from './demographic'; <- import if made
+
+const zVolunteer = z.object({
+  firstName: z.string(),
+  lastName: z.string(),
+  userName: z.string(),
+  email: z.string().email(),
+  phoneNumber: z.string(),
+//   events: zEvents.optional(),
```

Comment:
I see you are just waiting for the zEvent obj to be finished.

Reply:
I think I integrated it correctly based off of roleVerifications, let me know if you see anything off.